e2e_tests/test_workspace_creation.py

```diff
@@ -100,8 +100,10 @@ async def test_get_workspace_templates(template_name, token, verify) -> None:
         response = await client.get(
             f"https://{config.TRE_ID}.{config.RESOURCE_LOCATION}.cloudapp.azure.com{strings.API_WORKSPACE_TEMPLATES}",
             headers=headers)
-
-        assert (template_name in response.json()["templateNames"]), f"No {template_name} template found"
+        
+        all_templates = [value for item in response.json()["templates"]
```

Comment:
we should probably only check against names

Reply:
fixed